### DISCUSSION TITLE
e: Unable to Locate Package qt6-style-kvantum-themes

### DISCUSSION THREAD

**Discussion by ingaldo1** [tsujan/Kvantum#993] [Q&A]
Im using KDE Neon and when I want to install Kvantum with sudo apt install qt6-style-kvantum qt6-style-kvantum-themes
 I get that error e: Unable to Locate Package qt6-style-kvantum-themes. Any Idea? Thanks!

**Comment by tsujan** [comment]
You might want to ask this question in an Ubuntu forum. I doubt that Ubuntu maintainers check this page.

**Reply by tsujan** [reply]
By a quick search, I found https://ubuntuhandbook.org/index.php/2024/02/install-kvantum-ubuntu/. Apparently `qt6-style-kvantum-themes` doesn't exist (yet?), but don't rely on my words; I don't have Ubuntu.
